docs/integrations/databases/convex.mdx

```diff
@@ -3,8 +3,36 @@ title: Integrate Convex with Clerk
 description: Learn how to integrate Clerk into your Convex application.
 ---
 
+<TutorialHero
+  beforeYouStart={[
+    {
+      title: "Set up a Clerk application",
+      link: "https://clerk.com/docs/quickstarts/setup-clerk",
+    },
+    {
+      title: "Create a React + Convex application",
+      link: "https://docs.convex.dev/quickstart/react",
+    },
+  ]}
+>
+- Create a JWT Template based on Convex
+- Configure Convex with the Clerk issuer domain
+- Install and configure Clerk's React SDK
+- Configure the Clerk and Convex providers
+</TutorialHero>
+
 # Integrate Convex with Clerk
 
+Convex is a backend-as-a-service (BAAS) platform that allows you to build applications without having to worry about the backend. Convex provides a GraphQL API, authentication, and a database. Paired with Clerk's user authentication and management features, you can build a powerful application with minimal effort.
```

Comment:
Hey @alexisintech, I'm really curious, how did you come up with Convex providing a "GraphQL API"?
```suggestion
Convex is the full-stack TypeScript development platform. With Convex you get to build a backend with a provided realtime database, file storage, text search, scheduling and more. Paired with Clerk's user authentication and management features, you can build a powerful application with minimal effort.
```

Reply:
Hey @xixixao this sentence was actually given to me by GitHub Copilot (embarrassed) but this is exactly why I wanted your eyes on it! I've never worked with Convex, so I truly apologize for this mishap. I'll commit your suggestion now!